backend/app/api/v1/endpoints/presignedurl.py

```
@@ -0,0 +1,27 @@
+async def generate_presigned_url(request: Request, body: PresignedUploadRequest):
+    current_user = request.state.user   
+    file_upload_validator = FileService().validate(file_type=body.fileType, file_size=body.fileSize) # validate file type and size
+    
+    response = S3Client().generate_presigned_url(file_type=body.fileType)
+
+    return ResponseHandler().success_response(data=response, message="Presigned URL generated successfully")
+
+@router.post("/download")
+@auth_required()
+async def generate_download_url(request:Request, body:PresignedDownloadRequest):
+    current_user = request.state.user
```

Ruff (0.14.10)

Local variable `current_user` is assigned to but never used

Remove assignment to unused variable `current_user`

(F841)